jsonschema/validators.py

```diff
@@ -114,13 +120,15 @@ def _store_schema_list():
 
 
 def create(
-    meta_schema,
-    validators=(),
+    meta_schema: _typing.Schema,
+    validators: Mapping[str, _ValidatorCallback] | tuple[()] = (),
```

Comment:
Yep, I'm cool with that way of handling these!

I had some other ideas which I like less, but another option just occured to me.

With `_typing` we could do this:
```python
# in the definition
validators: Mapping[str, _ValidatorCallback] = _typing.EmptySequence

# in _typing
EmptySequence = typing.cast(typing.Any, ())
```

Because `Any` is a subtype of everything, the rules get wonky, and `mypy` and other checkers should be okay with this.
That way, `_typing.EmptySequence` can be used in any location where the empty tuple is used today.

Reply:
That seems also fine with me (although I think the fact that `()` works is more a quirk of `dict` than a generic empty sequence type, but I'll leave pedantry out until/unless it comes back to rear its head I think).

Reply:
Does it help that I also considered
```
EmptyMap = pyrsistent.freeze({})
```
?
😁 

I just know that this pattern occurs several times in the code, with `()` as an empty default. So it seems like it might be nice to sweep any messiness associated with it into `_typing`.